doc/030_preparing_a_new_repo.rst

```diff
@@ -381,6 +381,48 @@ established.
 .. _service account: https://cloud.google.com/storage/docs/authentication#service_accounts
 .. _create a service account key: https://cloud.google.com/storage/docs/authentication#generating-a-private-key
 
+Google Drive
+************
+
+Restic supports Google Drive as a backend.
+
+Restic connects to Google Drive via a `service account`_.
+
+To use the Google Drive backend, first `create a service account key`_
+and download the JSON credentials file to ``~/.config/restic/gdrive-account-key.json``.
+Second, create the backup folder in your Google Drive and share the folder with the aservice 
+account (use Service account ID as the email).
+
+Then you can use the ``gdrive:`` backend type to create a new repository under the folder
+you create in the Google Drive:
+
+.. code-block:: console
+
+    $ restic -r gdrive:restic-test/repo init
+    enter password for new backend:
+    enter password again:
+
+    created restic backend 9d34e4eec5 at gdrive:restic-test/repo
+    [...]
+
+The number of concurrent connections to the GCS service can be set with the
+`-o gs.connections=10`. By default, at most five parallel connections are
+established.
+
+Known limitations of the Google Drive backend:
+
+* Google Drive backend does not currently support concurrent backups to the
```

Comment:
What do you mean by that? Only one restic process can access a repo, not two restic processes running in parallel?

Reply:
Yes, correct, running two concurrent restic processes with the same repository can result in soft corruption of the repository. 

Google Drive allows duplicate file/folder names (names are just attributes on drive objects and do not have to be unique), so two restic processing can create two copies of the same data file, which will then confuse restore/check commands. It is possible to compensate for duplicate items during read, but I am not sure it's worth it yet (will certainly complicate the read logic).

As a side note, Drive may be creating duplicates even with a single restic process running. Not 100% sure, but I think this happened to me once during my testing and it's also mentioned in https://rclone.org/drive/#duplicated-files. So we may be forced to deal with duplicates. Really need more testing and user feedback on this one, I think.

Reply:
Ah, with restic, that should not be an issue: The repository format is designed in such a way that no two instances of restic will write the same file.

Reply:
Ah, cool, I assumed restic had some sort of check-before-write logic, but now I see it does something else. Config file is still a concern, I think, but not enough to deserve upfront warning. I'll remove the warning from the docs, but will keep the explanation in gdriveGetItem func.